qa/rolling-upgrade/src/test/resources/rest-api-spec/test/old_cluster/20_date_range.yml

```diff
@@ -42,7 +38,7 @@
 "Create index with joda style index that is incompatible with java.time (>6.1)":
   - skip:
       features: "allowed_warnings"
-      version: " - 6.8.0, 7.0.0 -"
+      version: " - 7.0.0 -"
```

Comment:
I think this test should be removed altogether. It looks like this `version` string is invalid. 

My understanding is that the old version string said that the test would be skipped before 6.8.0 and after 7.0.0. So, we can safely skip it for all OpenSearch versions.